pyproject.toml

```diff
@@ -40,7 +40,7 @@ dependencies = [
   "dill==0.3.8",
   "cloudpickle",
   "orjson>=3.10.5",
-  "pydantic>=2,<3",
+  "pydantic==2.8.2",
```

Comment:
Shouldn't it be '>2,<2.9'?

Reply:
I don't understand why we are so laissez-faire with dependency versions

Reply:
Exact pins are good for apps, but terrible for libraries.